internal/zinc-scripted/src/main/contraband-scala/sbt/internal/inctest/Project.scala

```diff
@@ -17,7 +17,7 @@ final class Project private (
     case _ => false
   }
   override def hashCode: Int = {
-    37 * (37 * (37 * (37 * (17 + name.##) + dependsOn.##) + in.##) + scalaVersion.##)
+    37 * (37 * (37 * (37 * (37 * (17 + "Project".##) + name.##) + dependsOn.##) + in.##) + scalaVersion.##)
```

Comment:
Okay. I suggest you give it a little bit of priority since it's a severe issue that could bite us.